### PR TITLE
Depenency modal fix

### PR DESCRIPTION
**Pull request by SaraVieira**
It was onSelect not onchange

### CODE REVIEW

**Comment by SaraVieira** [comment]
I did not touch the actions file, that was just prettier

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4925/commits/3203c272ca7fb6effde893925afae48fb112d116 is at https://pr4925.build.csb.dev/s/new.

**Comment by CompuIves** [comment]
Great! One thing, I got this when closing the modal:

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Explorer/Dependencies/AddDependency.tsx [hunk 1]
```diff
@@ -68,17 +68,18 @@ export const AddDependency: FunctionComponent<{ readonly?: boolean }> = () => {
   return (
     <>
       <Downshift
-        onChange={dependency => {
+        onSelect={dependency => {
           if (dependency === 'OPEN_MODAL') {
             modalOpened({ modal: 'searchDependencies' });
-          } else if (dependency) {
+            clearExplorerDependencies();
+          } else if (dependency?.name) {
```
Comment (SaraVieira) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Explorer/Dependencies/AddDependency.tsx:75]: @CompuIves Should be fixed with this

**Review by CompuIves — APPROVED** [review]
Nice!!

**Pull request merged**